cpp/src/command_classes/ThermostatMode.cpp

```diff
@@ -309,6 +363,14 @@ bool ThermostatMode::HandleMsg
 				}
 			}
 		}
+//		if ( GetVersion() >= 3)
+//		{
+//			ValueList::Item item;
+//			item.m_value = ThermostatMode_ManufacturerSpecific;
+//			item.m_label = c_modeName[item.m_value];
+//			m_supportedModes.push_back( item );
+//			Log::Write( LogLevel_Info, GetNodeId(), "    Added mode: %s", c_modeName[item.m_value] );
+//		}
```

Comment:
Consider not to add code, if it's not actually used :)

Reply:
Thanks for the PR BTW, looking forward to see this getting merged! 👍

Reply:
Thanks, forgot to remove that :see_no_evil: